components/ryr-instructions.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+
+const InstTitleStyle = { textAlign: 'center', marginBottom: '0.8em', fontSize: '1.8em' };
+
+const InstTypes = { textAlign: 'center', marginBottom: '1.5em' };
+const InstTypes2 = { textAlign: 'center', marginBottom: '1.5em', textIndent: '-4.9em' };
+
+const circleStyle = {
+  borderRadius: 50,
+  fontWeight: '970',
+  backgroundColor: '#06b700',
+  width: 100,
+  height: 100,
+  color: 'white',
+  padding: '3px 6px 3px 6px'
+};
+
+const NextStyle = {
+  marginBottom: '2em',
+  h3: '2em'
+};
+
+const RyrInstructions = props => {
+  let circleShape = props.size || circleStyle;
```

Comment:
There is a mismatch here. `prop.size` is a string and `circleStyle` is an object. As you are not using `prop.size` anywhere, you sjhould be able to get rid of this line.

Reply:
Fixed this issue